index.js

```diff
@@ -113,7 +72,13 @@ module.exports = function derbyBundler(app, options) {
         app._watchBundle(bundleFiles);
       }
       cb(null, source, map);
-    });
+    };
+    if (typeof backend.bundle !== 'function') {
+      // monkey patch in case where backend has been setup
+      // prior to being able to extend via prototype
```

Comment:
Per discussion over DMs, the actual reason was two separate copies of the `Backend` class due to a `npm link`. It could also happen for two different major versions.

Let's update this comment to accurately reflect the reason for the added defensiveness - anything that causes two copies of the `Backend` class